python/README.md

```diff
@@ -0,0 +1,6 @@
+# Benjamin's Kata
+
+# Python version
+`.python-version` holds the recommended version of python that the repo was developed in.
+
+It is recommended to us `pyenv` when developing on this repo.
```

Comment:
Reconsider phrase: you could make this clearer; "It is recommended to use `pyenv` to manage your installed python version when dev..."

Also typo: `us` -> `use`

Reply:
https://github.com/benjamincerigo/GildedRose-Refactoring-Kata/pull/1/commits/9989aa7bdefc60cabce5d1cb616e7e7b8c75f45f